### ISSUE TITLE
How to support OffsetDateTime filter

### ISSUE THREAD

**Issue by tawalisa** [json-path/JsonPath#613]
Hi ,

I want to use filter OffsetDateTime. But JsonPath has not supported it yet. I want to know if there is a way which we can extend this filter or JsonPath will support it in roadmap in the future?
```
OffsetDateTime offsetDateTime = OffsetDateTime.now();
        List valueAsList = new ArrayList<>();
        Map map1 = new LinkedHashMap();
        map1.put("MaximumValue",offsetDateTime);
        valueAsList.add(map1);
        Object ret = null;
        ret = JsonPath.parse(valueAsList).read("$[?]", Filter.filter(where("MaximumValue").gte(new Date())));
        System.out.println(ret);
```

We got this exception.
```
Exception in thread "main" com.jayway.jsonpath.JsonPathException: Could not determine value type
	at com.jayway.jsonpath.internal.filter.ValueNode.toValueNode(ValueNode.java:187)
	at com.jayway.jsonpath.Criteria.gte(Criteria.java:190)
```

**Comment by hezonghan** [comment]
Hi,
I'd like to try to help with this exciting feature!

**Comment by Youngea** [comment]
I would like to try to add this feature

**Comment by kallestenflo** [comment]
Fixed by #700
